src/main/java/com/mercuryCyclists/Sales/entity/Sale.java

```diff
@@ -26,6 +27,7 @@ public abstract class Sale {
     private Long quantity;
 
     @Column(name = "date_time")
+    @JsonFormat(pattern="dd/MM/yy")
```

Comment:
check my PR, probably better and easier to just set it to date.now()
happy with either way though tbh, but in reality we wouldn't let user set this

Reply:
I saw ur pr with the date.now() and I prefer it compared to how it is in this pr